### DISCUSSION TITLE
Can't run Perseus CLI on Windows

### DISCUSSION THREAD

**Discussion by VisualEhrmanntraut** [framesurge/perseus#5] [Q&A] [answered]
Hi, I'm trying to use perseus, but, I cannot run `perseus serve`:
![image](https://user-images.githubusercontent.com/30368284/132919660-52a0247b-70c4-4655-9e55-50d9b336e665.png)
But if I run `rollup` myself, it works fine:
![image](https://user-images.githubusercontent.com/30368284/132919719-a84f68f9-0534-4f0b-b84d-8e604523e070.png)

**Comment by arctic-hen7** [comment]
What shell are you running? Perseus tries to execute the command `rollup` directly on your system, without going through `cmd` or PowerShell, so it may be that Rollup isn't in the Windows equivalent of your `$PATH` somehow?

Also, I updated the title of the discussion because this should affect every command in the CLI. Thank you for bringing this to my attention!

**Reply by VisualEhrmanntraut** [reply]
I've not modified the path in my shell, I only just re-opened it after installing yarn, rollup and wasm-pack. And, it's powershell.

**Reply by arctic-hen7** [reply]
Hmm. Well, Rollup is the last prerequisite that Perseus checks, so that means that the others (`cargo` and `wasm-pack`) are working. That is extremely odd.

Does `rollup` work if you run it in `cmd` instead?

**Reply by VisualEhrmanntraut** [reply]
Yup
![image](https://user-images.githubusercontent.com/30368284/132939634-5c97316f-9a50-44df-aaa2-c5c98edcaef7.png)

**Reply by VisualEhrmanntraut** [reply]
![image](https://user-images.githubusercontent.com/30368284/132939666-472e0aa5-e68f-4971-b25a-4c2b65628512.png)
Hope this helps

**Reply by arctic-hen7** [reply]
Hmm. How about running `perseus serve` through `cmd`? That shouldn't change anything, but it might...

**Reply by VisualEhrmanntraut** [reply]
@arctic-hen7 I tried, it doesn't work either.

**Reply by VisualEhrmanntraut** [reply]
![image](https://user-images.githubusercontent.com/30368284/132940667-2b27ab53-af3e-47cd-a3da-7e17d69e9c57.png)

**Reply by arctic-hen7** [reply]
Okay, how about running `perseus serve` after setting the `PERSEUS_ROLLUP_PATH` variable to the absolute path of the `rollup` executable? That should work, and then we know the problem's in the resolution somehow.

**Reply by VisualEhrmanntraut** [reply]
> Okay, how about running `perseus serve` after setting the `PERSEUS_ROLLUP_PATH` variable to the absolute path of the `rollup` executable? That should work, and then we know the problem's in the resolution somehow.

It works

**Reply by VisualEhrmanntraut** [reply]
Well, @arctic-hen7 now I get this:
```
Couldn't execute command 'rm -rf dist/pkg'. Error was: 'The system cannot find the file specified. (os error 2)'.
```

**Reply by VisualEhrmanntraut** [reply]
rm -rf doesn't work on cmd or PowerShell

**Reply by VisualEhrmanntraut** [reply]
I can get past that using git bash, but it once again can't run rollup after

**Reply by VisualEhrmanntraut** [reply]
![image](https://user-images.githubusercontent.com/30368284/132943184-8b8aee32-76a6-40c5-a02e-348a05193c09.png)

**Reply by arctic-hen7** [reply]
Yeah that `rm` shouldn't be there at all. I'll push a fix ASAP. And I'll try to figure out why it can't find `rollup`.

**Reply by arctic-hen7** [reply]
Okay, I've released this and a number of other fixes in v0.1.4, would you mind trying that for the CLI and seeing if it works for you?

**Reply by VisualEhrmanntraut** [reply]
It works

**Reply by arctic-hen7** [reply]
Fantastic! Happy to help, and sorry for the inconvenience!

**Comment by VisualEhrmanntraut** [comment, marked as answer]
Resolved in v0.1.4: https://github.com/arctic-hen7/perseus/discussions/5#discussioncomment-1311270
